backend/lib/uuid/BUILD

```diff
@@ -1,5 +1,7 @@
 load("@rules_java//java:defs.bzl", "java_library")
 
+# Due to a bug in the abbreviation checkstyle rule
```

Comment:
For some reason I cannot add `UUID` or `UUIDv5` to the list of ignored abbreviations, so this is a workaround